hexrd/ui/image_canvas.py

```diff
@@ -109,6 +111,7 @@ def clear_azimuthal_integral_axis(self):
         self.clear_wppf_plot()
         self.azimuthal_integral_axis = None
         self.azimuthal_line_artist = None
+        self.azimuthal_overlay_artists.clear()
```

Comment:
See other comment about creating a `clear_azimuthal_overlay_artists()` function. We would then call that here.

Reply:
Updated.